src/com/lftechnology/java/training/alina/collections/generics/instantiateproduct/ProductDemo.java

```diff
@@ -0,0 +1,86 @@
+
+package com.lftechnology.java.training.alina.collections.generics.instantiateproduct;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Create a generic class Product with a display method and instantiate an
+ * object of Product using different data types. Eg : Product<String,String> ,
+ * Product<String,Integer>
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class ProductDemo {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)